backend/internal/model/game_test.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func TestResetScore(t *testing.T) {
-	game := &Game{gameScore{3, 1}}
+	game := &game{gameScore{3, 1}}
```

Comment:
Some would say that you should only test public interface, but this is a valid approach too